ambry-mysql/src/main/java/com/github/ambry/accountstats/PartitionClassReportsDao.java

```diff
@@ -98,6 +98,18 @@
       AGGREGATED_PARTITION_CLASS_REPORTS_TABLE, ID_COLUMN, PARTITION_CLASS_NAMES_TABLE, CLUSTER_NAME_COLUMN,
       NAME_COLUMN, STORAGE_USAGE_COLUMN, UPDATED_AT_COLUMN);
 
+  /**
+   * eg : DELETE FROM AggregatedPartitionClassReports
+   *      WHERE
+   *      PartitionClassId IN (SELECT id FROM PartitionClassNames WHERE clusterName = "ambry-test" AND name = "default")
+   *      AND accountId = 101
+   *      AND containerId = 201;
+   */
+  private static final String deleteAggregatedSql =
+      String.format("DELETE FROM %s WHERE %s IN (SELECT %s FROM %s WHERE %s=? AND %s=?) AND %s=? AND %s=?",
```

Comment:
Just to confirm, in `PartitionClassNames` table, if two partition classes have same name but in different clusters, they will have two different ids, correct?

Reply:
that's right, the `PartitionClassNames` has a auto incremented id and (clusterName, partitionClassName) form an unique key in the table.